models/class.metrilo-order-data.php

```diff
@@ -16,11 +16,18 @@ public function get_orders($chunk_id)
     {
         $orders = [];
         $order_ids = $this->db_connection->get_col(
-            "SELECT id FROM {$this->db_connection->posts}
-            WHERE post_type = 'shop_order'
-            ORDER BY id ASC
-            limit {$this->chunk_items}
-            offset {$chunk_id}"
+            $this->db_connection->prepare(
+        "
+                SELECT id
+                FROM {$this->db_connection->posts}
```

Comment:
I see that you used `{...}` here and nowhere else. What's the best practice?

Reply:
I used curly brackets to all queries, but the examples i found about preparing the query were all without brackets, so i removed them. Theese are leftovers and should be removed, as other query strings are without them.